ibtk/include/ibtk/config.h.in

```diff
@@ -78,6 +78,7 @@ _Pragma("GCC diagnostic ignored \"-Wdeprecated-copy\"")         \
 _Pragma("GCC diagnostic ignored \"-Wunused-parameter\"")        \
 _Pragma("GCC diagnostic ignored \"-Wparentheses\"")             \
 _Pragma("GCC diagnostic ignored \"-Wdangling-reference\"")      \
+_Pragma("GCC diagnostic ignored \"-Warray-bounds\"")            \
```

Comment:
Should this be in this PR?

Reply:
Technically no, I can move it somewhere else.